externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/WithSparkSQLEngine.scala

```diff
@@ -39,14 +44,18 @@ trait WithSparkSQLEngine extends KyuubiFunSuite {
 
     SparkSession.clearActiveSession()
     SparkSession.clearDefaultSession()
+    SparkSQLEngine.kyuubiConf.loadFromSystemProperties()
```

Comment:
Add this code to ensure the system properties has been loaded into kyuubi conf, otherwise the same jvm would just load once at first time.